src/app/android/.gitignore

```diff
@@ -11,3 +11,8 @@ GeneratedPluginRegistrant.java
 key.properties
 **/*.keystore
 **/*.jks
+
+# Android Studio will place build artifacts here.
```

Comment:
just curious, is it recommended to do it like this, or did you just find it more clear to do it this way.

Reply:
Ohh, it was already like that when creating the Flutter project, I think it also helps, to divide stuff, per platform, perf project and global.